lib/mix/test/mix/cli_test.exs

```diff
@@ -169,8 +171,9 @@ defmodule Mix.CLITest do
     System.delete_env("MIX_EXS")
   end
 
-  test "new with tests and cover" do
-    in_tmp("new_with_tests", fn ->
+  @tag tmp_dir: "new_with_tests"
```

Comment:
this is the test where customising the path was useful, by having the final directory as we want, we can simply do: `mix new .` below.

Reply:
Oh, I see. I understand it then. Let's keep it as is!